app/models/position.rb

```diff
@@ -16,7 +16,9 @@ class Position < ActiveRecord::Base
   #  obj.save
   #end
 
-  scope :from_role, -> role { joins(:role).where(acl_role: { acl_role_name: role })}
+  scope :from_role, -> role { joins(:role).where(acl_role: { acl_role_name: role }) }
+
+  scope :for_phonebook, -> { joins(:role, :user).where('acl_role.acl_role_id NOT IN (34,21,37,5,31,35,36)').where('user.user_active = 1') }
```

Comment:
Line is too long. [139/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.